app/src/main/java/com/example/findu/presentation/ui/search/detail/SearchDisappearDetailFragment.kt

```
@@ -58,12 +76,46 @@ class SearchDisappearDetailFragment : Fragment() {
+                    naverMap?.moveCamera(cameraUpdate)
+
+                    val marker = Marker().apply {
+                        position = location
+                        map = naverMap
+                        icon = OverlayImage.fromResource(R.drawable.ic_search_map_marker)
+                        height = 23
+                    }
+                    marker.position = location
+                    marker.map = naverMap
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)